AUTHORS

```diff
@@ -228,6 +231,7 @@ Lorrany Azevedo <lorrany90@gmail.com>
 Lucklita Theng <laklitatheng@gmail.com>
 Luis Ulloa <scssaluis@gmail.com>
 Mahendra Suthar <mahendra.suthar.1290@gmail.com>
+Mai Elshiashi <mai.elshiashi@gmail.com>
```

Comment:
I think this is the same person, it's fine to delete the line below this one.

Reply:
Done